Source/CombatExtended/CombatExtended/Comps/HediffComp_InfecterCE.cs

```diff
@@ -11,22 +11,53 @@ namespace CombatExtended
     public class HediffComp_InfecterCE : HediffComp
     {
         private const float infectionInnerModifier = 3f;                            // Injuries to inner body parts will be this much more likely to get infected
-        private static readonly IntRange InfectionDelayHours = new IntRange(6, 12); // Infections will appear somewhere within this timeframe
+        //private static readonly IntRange InfectionDelayHours = new IntRange(6, 12); // Infections will appear somewhere within this timeframe
```

Comment:
What's with these changes?

Reply:
There were some compilation errors in here, and I was unsure which part was custom and which was not. I was not able to fix it fast enough since you know your own code better than I do, I focused on other errors, and replaced it with the updated Rimworld code it was based on for now.

Reply:
Oh my bad, sorry.
I will revert it, but the Method tendTick of HediffComp_TendDuration is missing,
what do you suggest to fix it?
[HediffComp_TendDuration compTended = parent.TryGetComp<HediffComp_TendDuration>();]

Reply:
tendTicksLeft would be available, but it would need a complete refactor of that part. Unless I am missing something of course.

Reply:
Also, I would suggest taking a look at
https://github.com/josh-m/RW-Decompile/blob/master/Verse/HediffComp_Infecter.cs
to make sure nothing important in terms of feature or balancing got added.
InfectionChanceFactorFromTendQualityCurve and
InfectionChanceFactorFromSeverityCurve for example.

Reply:
The issue here is we need to calculate the amount of time a hediff was untended. Since the exact tick it was tended is no longer available we'd need to either recreate this functionality or infer it from the existing variables. Easiest way to handle it would probably to add a counter class variable and increase it in CompPostTick if the hediff is untended.

Reply:
You know the Rimworld code better and can implement it faster, I would have to figure out what exactly tendTicksLeft does, without any documentation.

Reply:
Okay pushed a potential fix but I am not sure if that works, if not then you should implement it.